ndc-oslo-2016/category.json

```diff
@@ -0,0 +1,5 @@
+{
+  "alias": "category/NDC-Oslo-2016",
```

Comment:
You can drop this key as aliases are deprecated. Thanks for the proactive work though.